docs/networking.rst

```diff
@@ -82,5 +82,46 @@ The configuration for this architecture requires the following settings:
 With use_public_ips set to false The VPC must be correctly setup to use the Proxy for all traffic.
 Web access is required for both Master and Compute instances.
 
+.. _awsbatch_networking:
+
+AWS ParallelCluster with awsbatch scheduler
+-------------------------------------------
+
+When using ``awsbatch`` as scheduler type, ParallelCluster creates an AWS Batch Managed Compute Environment which takes
+care of managing ECS container instances, launched in the ``compute_subnet``. In order for AWS Batch to function
+correctly, Amazon ECS container instances need external network access to communicate with the Amazon ECS service
+endpoint. This translates into the following scenarios:
+
+* The ``compute_subnet`` uses a NAT Gateway to access the Internet. (Recommended approach)
+* Instances launched in the ``compute_subnet`` have public IP addresses and can reach the Internet through an
+  Internet Gateway.
+
+Additionally, according to AWS Batch docs:
```

Comment:
I'd add something like: _Additionally, If you are interested on multi-node-parallel jobs ...._ (but it is my opinion, feel free to let whatever you like)

Reply:
I like it!